src/site/content/en/lighthouse-best-practices/errors-in-console/index.md

```diff
@@ -2,82 +2,75 @@
+Most browsers ship with built-in developer tools.
+These developer tools usually include a [console](https://developers.google.com/web/tools/chrome-devtools/console/).
+The console gives you information about the page that's currently running.
+
+Messages logged in the console have a severity level:
+either `Verbose`, `Info`, `Warning`, or `Error`.
+An `Error` message means there's a problem on your page that you need to resolve.
+
+## How the Lighthouse browser error audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags all browser errors logged to the console:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
-  <figcaption class="w-figcaption">
-    Figure 1. Browser errors logged to the console.
-  </figcaption>
+  <img class="w-screenshot" src="errors-in-console.png" alt="Lighthouse audit showing browser errors in the console">
 </figure>
 
-## How this audit fails
+{% include 'content/lighthouse-best-practices/scoring.njk' %}
 
-Most browsers ship with built-in developer tools.
-These developer tools usually include a console.
-The console gives you information about a page while that page is running.
-Ultimately,
-the messages you see in the console either come from the web developers who built the page,
-or the browser itself.
-
-When someone logs a message to the console,
-they can indicate the importance, or "severity level" of the message.
-An "error" message is an important message
-representing an unresolved failure in the page.
-In other words,
-when you see an error, the page isn't running as intended.
+## How to fix browser errors
+
+Fix each browser error that Lighthouse reports
+to ensure that your page runs as expected for all your users.
+
+Chrome DevTools includes a couple tools
+to help you track down the cause of errors:
+
+- Below the text of each error, the console shows the
+  [call stack](https://en.wikipedia.org/wiki/Call_stack)
+  that caused the problematic code to execute.
+- A link at the top-right of each error shows you the code
+  that caused the error.
+
+For example, this screenshot shows a page with two errors:
 
 <figure class="w-figure">
   <img class="w-screenshot w-screenshot--filled" src="errors.png" alt="An example of console errors in Chrome DevTools">
-  <figcaption class="w-figcaption">
-    Figure 2. An example of console errors in Chrome DevTools.
-  </figcaption>
 </figure>
 
-**Figure 2** shows two errors.
-The top one comes from a web developer,
+In the example above, the first error comes from a web developer,
 via a call to
 [`console.error()`](https://developers.google.com/web/tools/chrome-devtools/console/console-reference#error).
-The bottom one comes from the browser,
-which indicates that a variable used in one of the page's scripts does not exist.
-Lighthouse flags the browser errors.
```

Comment:
As mentioned above, if we're going to conceptually explain the Console then I do think it's key to explain that some messages come from the people who built the page, and other messages are coming from the browser.

Reply:
Yeah, as above, I was worried about how long the post was getting. Also, there was a bit of screwiness with the flow, which I tried to fix.